### ISSUE TITLE
Removal of 300 seconds expiry time tests from the automated test suite

### ISSUE THREAD

**Issue by AndrewGuiseley** [nhsconnect/gpconnect-provider-testing#285]
As agreed, please can the following 2 tests be removed from the automated test suite:

Expiry Time - greater than 300 seconds
Expiry Time - less than 300 seconds

Providers should only be validating that the exp claim value (i.e. that this time has not yet passed)

**Comment by jonnyry** [comment]
Just to mention the demonstrator intentionally continues to perform these checks (as additional consumer validations), whereas the provider systems are not expected to.

**Comment by Sado1234** [comment]
Hi @jonnyry what are your thoughts, instead of deleting we place these type of tests in a 'DemonstratorFeature' file to continue checking the Demonstrator functionality?

**Comment by PeterGresty** [comment]
DemonstratorOnly.feature maybe - for even more clarity

**Comment by jonnyry** [comment]
Yes sounds like a reasonable idea.  So long as the providers are clear they don't need to run the tests.
